cdap-client-tests/src/main/java/co/cask/cdap/client/app/AppReturnsArgs.java

```diff
@@ -49,7 +50,8 @@ public void configure() {
     @Path(ENDPOINT)
     @GET
     public void handler(HttpServiceRequest request, HttpServiceResponder responder) {
-      responder.sendJson(200, getContext().getRuntimeArguments());
+      TreeMap<String, String> values = new TreeMap<>(getContext().getRuntimeArguments());
```

Comment:
@chtyim @gsps1 I will remove the TreeMap. I had changed the assert to compare maps instead of JSON strings.